pyne/mesh.py

```diff
@@ -1433,6 +1433,20 @@ def tag_cell_fracs(self, cell_fracs):
                  size=1, dtype=float)
 
 class StatMesh(Mesh):
+    """This class extends the basic Mesh class by modifying the standard
+    mathematical operations that are performed on multiple meshes.
+
+    A StatMesh assumes that each value being operated upon also has a
+    statistical error associaed with it, and forces operations on the
+    statistical error as well.
+
+    For example, when to quantities are added together, c = a + b, the
+    statistical error of c is found by combining the statistical errors of a
+    and b.
+
```

Comment:
We may want a note in here about how it finds error values associated with data ("`tag name` + `_rel_error`").

Reply:
done